app/src/main/java/org/kiwix/kiwixmobile/zimManager/ZimManageViewModel.kt

```diff
@@ -286,10 +286,14 @@ class ZimManageViewModel @Inject constructor(
     val booksFromDao = books().asFlowable()
     val networkLibrary = PublishProcessor.create<LibraryNetworkEntity>()
     val languages = languageDao.languages().asFlowable()
+
+    viewModelScope.launch {
```

Comment:
This method no longer exists, so you should put this inside the `observeCoroutineFlows` method.